daemon/cmd/daemon_main.go

```diff
@@ -1163,6 +1163,10 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	if len(option.Config.Devices) != 0 && option.Config.EnableIPSec {
+		log.Fatal("Devices cannot be used with IPSec.")
```

Comment:
Given we have several options that set devices, maybe we should explicitly state to which we're referring?
```suggestion
		log.Fatalf("--%s cannot be used with IPSec.", option.Devices)
```

Reply:
Sounds like a good idea 👍

Reply:
I've updated the PR.